index.html

```diff
@@ -4403,20 +4415,55 @@ <h2>Canonicalization</h2>
   <span class="rfc2119-assertion" id="processors-preserve-prefixes">
   In order to support canonicalization, implementations 
   of TD Processors MUST preserve the prefixes used for extension vocabularies.</span>
-  This includes preserving and expressing a vocabulary term as a URL if no prefix was used.
+  This includes preserving and expressing a vocabulary term as a URL 
+  if no prefix is defined for that URL.
+  However, we need to
+  support stable reserialization in TD Processors that expand all
+  prefixes into URLs and then need to collapse them back to prefixes
```

Comment:
we need to grantee that normalized prefixes are used

Reply:
yes, maintaining prefixes is a TD processor requirements (this includes directories).  This can be done though by treating the @context as a regular object...

Reply:
I *think* this was already taken care of in the draft, please confirm and resolve the conversation if so.